sufia-models/lib/sufia/models/utils.rb

```diff
@@ -14,6 +28,7 @@ def retry_unless(number_of_tries, condition, &block)
         number_of_tries.each do
           result = block.call
           return result unless condition.call
+          sleep(Sufia.config.retry_unless_sleep) if Sufia.config.retry_unless_sleep > 0
```

Comment:
not necessarily.   Certainly people can do things like expect(sleep) and our new sleep might trip that up.  It's mostly for testing, but I do not see the harm...